docs/CHANGELOG.asciidoc

```diff
@@ -54,6 +54,12 @@ To review the breaking changes in previous versions, refer to the following:
 
 {kibana-ref-all}/8.2/release-notes-8.2.0.html#breaking-changes-8.2.0[8.2.0] | {kibana-ref-all}/8.1/release-notes-8.1.0.html#breaking-changes-8.1.0[8.1.0] | {kibana-ref-all}/8.0/release-notes-8.0.0.html#breaking-changes-8.0.0[8.0.0] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc2.html#breaking-changes-8.0.0-rc2[8.0.0-rc2] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc1.html#breaking-changes-8.0.0-rc1[8.0.0-rc1] | {kibana-ref-all}/8.0/release-notes-8.0.0-beta1.html#breaking-changes-8.0.0-beta1[8.0.0-beta1] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha2.html#breaking-changes-8.0.0-alpha2[8.0.0-alpha2] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha1.html#breaking-changes-8.0.0-alpha1[8.0.0-alpha1]
 
+[discrete]
+=== Known issues
+
+* Elasticsearch API commands will fail if you add any URL arguments (e.g. `?v`, `?pretty`) to the call. 
+This only impacts the Kibana Dev Tools.  All other sources of API calls are not affected (e.g. curl, Elastic Cloud API console) {kibana-issue}133965[#133965]
```

Comment:
Likewise, let's change "API calls" to "API requests."

Reply:
Done. thx!